deploy_pulp3/pulp3/tasks/main.yml

```diff
@@ -0,0 +1,85 @@
+---
+# tasks file for pulp3
+- name: Create user {{ pulp_user }}
+  user:
+    name: "{{ pulp_user }}"
+    comment: "Pulp user"
+
+- name: Get user's home directory
+  shell: 'getent passwd "{{ pulp_user }}" | cut -d: -f6'
```

Comment:
There is a [getent](http://docs.ansible.com/ansible/latest/getent_module.html) module which could achieve this in a much more readable manner, something like this:
```
- name: Get user's home directory
    getent:
     database: passwd
     key: "{{ pulp_user }}"
     split: ':'
- debug:
   var: getent_passwd['{{ pulp_user }}'][4]
```